homeassistant/components/ozw/lock.py

```diff
@@ -37,3 +75,51 @@ async def async_lock(self, **kwargs):
     async def async_unlock(self, **kwargs):
         """Unlock the lock."""
         self.values.primary.send_value(False)
+
+    @callback
+    def async_set_usercode(self, code_slot, usercode):
+        """Set the usercode to index X on the lock."""
+        lock_node = self.values._node.values()  # pytest: disable=protected-access
```

Comment:
`self.values.primary.node` should also give you the node. I guess updating `values` to actually expose `node` would make more sense.